packages/shared/providers.yaml

```diff
@@ -727,6 +747,8 @@ battlenet:
         response_type: code
     token_params:
         grant_type: authorization_code
+    proxy:
+        base_url: https://${connectionConfig.apiBaseUrl}
```

Comment:
could we always put `.api.blizzard.com`?

Reply:
It may hugely vary depending on the region. For regions outside of China, the base url follows the format `{region}.api.blizzard.com`, where `{region}` can be `us`, `eu`, `kr`, or `tw`. For China, the  api base url is `gateway.battlenet.com.cn`.

Reply:
I see, can it be named `apiDomain` then or you also expect some paths?

Reply:
I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/3155/commits/5114e7cb33009b4b0ff2259cc8aec1ec949d3073). Yes after the API domain we can now have the endpoint url `
/data/wow/achievement/index`